logos/README.md

```diff
@@ -0,0 +1,35 @@
+# TOML logos
+
+## Logos and marks
+
+TOML's graphic material consists of the logo showing the name TOML in full and the mark showing only the letter T.
+The logo was designed to be used wherever display space is not limited.
+The mark was designed to be used in place of the logo when display space is limited or when it specifically refers to a TOML file.
+
+The color versions are intended for a white background, while the reverse versions are intended for a black background. 
+The fallback versions were designed for when the background cannot be controlled, such as in Github `readme.md` files where the background depends on the user's dark or light theme preferences.
+
+![toml-logos-list](./show/logos-list.png)
+
+## Icons
+
+The icons are indented to be used as favicon.
+
+![toml-icon-list](./show/icon-list.png)
+
+## Tagline
+
+The tagline "config files for human" is intended to be use in conjuction with the logo on headers, footers, or any other place where the logo can be displayed in large format (to preserve the lisibility of the tagline).
```

Comment:
The tagline "config files for humans" is intended to be use in conjuction with the logo on headers, footers, or any other place where the logo can be displayed in large format (to preserve the lisibility of the tagline).